draft-ietf-quic-invariants.md

```diff
@@ -118,8 +118,9 @@ This document uses terms and notational conventions from {{QUIC-TRANSPORT}}.
 
 # Notational Conventions
 
-Packet diagrams in this document use a format defined in {{QUIC-TRANSPORT}} to
-illustrate the order and size of fields.
+Packet diagrams in this document use a format taken from {{QUIC-TRANSPORT}} to
+illustrate the order and size of fields.  That format is defined in this
+section.
```

Comment:
```suggestion
The format used by packet diagrams in this document is defined in this
section, and is the same as that used in {{QUIC-TRANSPORT}}.
```

Reply:
```suggestion
The format of packets is described using the notation defined in this section.
This notation is the same as that used in {{QUIC-TRANSPORT}}.
```

Reply:
@martinthomson : your suggestion works for me.